shared/subprocess/proc.go

```diff
@@ -41,6 +41,10 @@ type Process struct {
 }
 
 func (p *Process) hasApparmor() bool {
+	if os.Getenv("LXD_SECURITY_APPARMOR") == "false" {
```

Comment:
How about we use `shared.IsFalse`?

Reply:
I did think about that, but the existing usage of `os.Getenv("LXD_SECURITY_APPARMOR") ` compares to "false" so I didn't want to introduce a different type of logic here.

Reply:
OK yeah, I wasn't sure if there was some historical context for that particularly strict check.

Reply:
Fixed